src/linter/cache.go

```diff
@@ -44,8 +44,7 @@ import (
 //     46 - changed the way of inferring the return type of functions and methods
 //     47 - forced cache version invalidation due to the #921
 //     48 - renamed meta.TypesMap to types.Map; this affects gob encoding
-//     49 - added WithNull field to FuncInfoOverride structure
-const cacheVersion = 49
+const cacheVersion = 47
```

Comment:
Why 47?

Reply:
It looks like this is a bug from the master. 
Let's increase the version to a separate PR.